Tasks/ServiceFabricDeploy/task.json

```diff
@@ -384,8 +384,9 @@
         "ItemSearchMoreThanOneFound": "Found more than one item with search pattern {0}. There can be only one.",
         "ItemSearchNoFilesFound": "No items were found with search pattern {0}.",
         "SearchingForPath": "Searching for path: {0}",
-        "FoundPath": "Found path: {0}",
         "PathDoesNotExist": "Path '{0}' does not exist.",
+        "FoundPath": "Found path: {0}",
+        "InvalidApplicationName": "Application name '{0}' is invalid",
```

Comment:
Consider: If you're only checking empty string, why give such a generic error?

Reply:
+1. Just say that application name should be a non-empty string.